src/Integration.Vsix.UnitTests/Analysis/IssueHandlerTests.cs

```diff
@@ -79,30 +79,19 @@ public void HandleNewIssues_UpdatedSnapshotHasExpectedValues()
         }
 
         [TestMethod]
-        public void HandleNewIssues_IssuesGetMatchesIsCalled()
+        public void HandleNewIssues_ClientSuppressionSynchronizerIsCalled()
         {
-            var inputIssues = new[]
-            {
-                CreateIssue("S111", startLine: 1, endLine: 1),
-                CreateIssue("S222", startLine: 2, endLine: 2)
-            };
-
             var notificationHandler = new SnapshotChangeHandler();
-            var issuesFilter = new Mock<IIssuesFilter>();
-
+            var clientSuppressionSynchronizer = new Mock<IClientSuppressionSynchronizer>();
 
-            var testSubject = CreateTestSubject(notificationHandler.OnSnapshotChanged, issuesFilter.Object);
+            var testSubject = CreateTestSubject(notificationHandler.OnSnapshotChanged, clientSuppressionSynchronizer.Object);
 
             // Act
-            testSubject.HandleNewIssues(inputIssues);
+            testSubject.HandleNewIssues(new List<IAnalysisIssueVisualization>());
 
             // Assert
-            issuesFilter.Verify(x => x.GetMatches(inputIssues), Times.Once);
+            clientSuppressionSynchronizer.Verify(x => x.SynchronizeSuppressedIssues(), Times.Once);
             notificationHandler.InvocationCount.Should().Be(1);
-
-            // Check the updated issues
-            notificationHandler.UpdatedSnapshot.Issues.Count().Should().Be(2);
-            notificationHandler.UpdatedSnapshot.Issues.Should().BeEquivalentTo(inputIssues);
```

Comment:
We dont need to check the issue count as we aren't changing the incoming list of issues anywhere